Updating meetups form layout (fixes #1786)

BhargaviNadendla opened the pull request.

paulbert commented on src/app/meetups/add-meetups/meetups-add.component.ts:
```diff
@@ -16,7 +16,8 @@ import { switchMap } from 'rxjs/operators';
 @Component({
-  templateUrl: './meetups-add.component.html'
+  templateUrl: './meetups-add.component.html',
+  styles: [ 'mat-radio-button {margin:10px;}', 'mat-form-field.half-width {width:205pt;}' ]
```
A few things here:
* Please reformat these styles to match other components, like:
```
styles: [ `
  mat-radio-button {
    margin:10px;
  }
` ]
```
* That `half-width` class is misleading since it hard codes what half width is rather than using a value relative to its parent.
* In the future, please do not use the `pt` unit and instead use `px` or `rem`

paulbert commented on src/app/meetups/add-meetups/meetups-add.component.html:
```diff
@@ -8,69 +8,67 @@
       </div>
       <div>
-        <label i18n>Recurring</label>
+        <label i18n>Recurring Frequency  </label>
```
Remove extra spaces between the text and the end tag

paulbert commented on src/app/meetups/add-meetups/meetups-add.component.html:
```diff
@@ -8,69 +8,67 @@
-          <mat-radio-button value="daily" (change)="toggleDaily($event.value, false)" i18n>Daily</mat-radio-button>
-          <mat-radio-button value="weekly" (change)="toggleDaily($event.value, true)" i18n>Weekly</mat-radio-button>
-          <mat-radio-button value="none" (change)="toggleDaily($event.value, false)" i18n>None</mat-radio-button>
+          <mat-radio-button value="daily" (change)="toggleDaily($event.value, false)" il8n>Daily  </mat-radio-button>
```
Remove extra spaces between the text and the end tag

paulbert commented on src/app/meetups/add-meetups/meetups-add.component.html:
```diff
@@ -8,69 +8,67 @@
-          <mat-radio-button value="weekly" (change)="toggleDaily($event.value, true)" i18n>Weekly</mat-radio-button>
-          <mat-radio-button value="none" (change)="toggleDaily($event.value, false)" i18n>None</mat-radio-button>
+          <mat-radio-button value="daily" (change)="toggleDaily($event.value, false)" il8n>Daily  </mat-radio-button>
+          <mat-radio-button value="weekly" (change)="toggleDaily($event.value, true)" i18n>Weekly  </mat-radio-button>
```
Remove extra spaces between the text and the end tag

paulbert commented on src/app/meetups/add-meetups/meetups-add.component.html:
```diff
@@ -8,69 +8,67 @@
-          <mat-radio-button value="none" (change)="toggleDaily($event.value, false)" i18n>None</mat-radio-button>
+          <mat-radio-button value="daily" (change)="toggleDaily($event.value, false)" il8n>Daily  </mat-radio-button>
+          <mat-radio-button value="weekly" (change)="toggleDaily($event.value, true)" i18n>Weekly  </mat-radio-button>
+          <mat-radio-button value="none" (change)="toggleDaily($event.value, false)" i18n>None  </mat-radio-button>
```
Remove extra spaces between the text and the end tag

paulbert commented on src/app/meetups/add-meetups/meetups-add.component.html:
```diff
@@ -8,69 +8,67 @@
+          <mat-radio-button value="none" (change)="toggleDaily($event.value, false)" i18n>None  </mat-radio-button>
           <mat-error><planet-form-error-messages [control]="meetupForm.controls.recurring"></planet-form-error-messages></mat-error>
-        </mat-radio-group>
+        </mat-radio-group><br>
```
This isn't a good use of the `<br>` tag (it should only be used for poetry & addresses), and the form looks good without this extra break.

In the future, use styles to add spacing between elements instead.

BhargaviNadendla commented:
@paulbert  Updated all the changes, but couldn't find a built in class e.g., "half-width" to increase the mat input field to 50% unlike the `class="full-width"` does it to full. So, included a custom style class `custom-input-width` instead

lmmrssa commented:
@BhargaviNadendla form looks better than before. Could you make the class to be more generic like half-width or something similar. Also there are `div` which are not used anymore. Also could you see if you can make add spacing between inputs

BhargaviNadendla commented:
@lmmrssa 1. Updated according to the changes requested, but as @paulbert suggested, the class name "half-width" is a bit misleading as I am defining the width explicitly because I couldn't find any built in class to set the width to 50%. So, defined the width as "custom-input-width" class. 

2. Unused <div> are removed except the one that is covering start time and end time and added spaces between the inputs

paulbert commented:
I'm going to close this.  It's out of date & I'd like to move to form layouts that are more flexible and responsive
